crs_support/worlds/crs.world

```diff
@@ -11,7 +11,25 @@
     </include>
 
     <gravity>0 0 0</gravity>
-
+    <include>
+		<plugin name="gazebo_ros_joint_pose_trajectory"
+		     filename="libgazebo_ros_joint_pose_trajectory.so">
+		   <ros>
+		     <namespace>/crs</namespace>
+		   </ros>
+		   <update_rate>2</update_rate>
```

Comment:
This rate should probably be higher, otherwise I believe we can only update robot positions at 2 Hz

Reply:
Also I'm not entirely sure this is required as it is also included in the urdf